### ISSUE TITLE
Effort stats

### ISSUE THREAD

**Issue by xaur** [artikozel/decred-translations#17]
* who: @Guang168
* what: Blockchain governance: how Decred iterates upon Bitcoin ([source](https://medium.com/decred/blockchain-governance-how-decred-iterates-upon-bitcoin-3cc7030c655e))
* to: Chinese ([link]())
* time: 3+ hours ([report](https://matrix.to/#/!lbzTjhzNbIaDbuAxkS:decred.org/$154505941244638HLgAi:decred.org))

* who: elian
* what: Decred Journal - November 2018 ([source](https://xaur.github.io/decred-news/journal/201811.html))
* to: Spanish ([link](https://medium.com/@decred_es/revista-decred-noviembre-2018-a3e52c5fc1a9))
* time: (4h) to rephrase and change words after google translate, which does like 70% of the job, the other 30% is about giving soul to the text otherwise feel cold and soulless ([report](https://matrix.to/#/!lbzTjhzNbIaDbuAxkS:decred.org/$154505884144636rxmQN:decred.org))

* who: @artikozel
* what: Decred Journal - November 2018 ([source](https://xaur.github.io/decred-news/journal/201811.html))
* to: Polish ([link](https://github.com/artikozel/DecredJournalPL/blob/master/journal/201811_DecredJournalPL.md))
* time: 10 hours, estimated ([estimate](https://matrix.to/#/!lbzTjhzNbIaDbuAxkS:decred.org/$154524214446320jMUNk:decred.org))

* who: @MaiconJunge
* what: Decred Journal - December 2018 ([source](https://github.com/xaur/decred-news/blob/master/journal/201812.md)), full version
* to: Portuguese [link](https://github.com/MaiconJunge/decred-news/blob/master/journal/201812PTBR.md)
* time: 2 days to translate and 2 to format ([report](https://matrix.to/#/!lbzTjhzNbIaDbuAxkS:decred.org/$154879391221353bYErK:decred.org))

Template:

```
* who: {github or chat name}
* what: {article name on source lang} ([source]({link to source text}))
* to: {target lang} ([link]({link to translated text}))
* time: {how long it took} ([report]({link to chat message}))
```

**Comment by xaur** [comment]
What do you mean by implement?

**Comment by xaur** [comment]
```
* who: @Guang168
* what: Decred Journal - December 2018 ([source](https://xaur.github.io/decred-news/journal/201812.html))
* to: Chinese ([link](https://github.com/Guang168/DecredCNJournal/blob/master/201812_DecredJournalCN.md))
* time: 9 hours ([report](https://matrix.to/#/!lbzTjhzNbIaDbuAxkS:decred.org/$154502025444491kdLiq:decred.org))
```

@Guang168 can you please comment whether it was full or partial translation?

**Comment by Guang168** [comment]
> ```
> * who: @Guang168
> * what: Decred Journal - December 2018 ([source](https://xaur.github.io/decred-news/journal/201812.html))
> * to: Chinese ([link](https://github.com/Guang168/DecredCNJournal/blob/master/201812_DecredJournalCN.md))
> * time: 9 hours ([report](https://matrix.to/#/!lbzTjhzNbIaDbuAxkS:decred.org/$154502025444491kdLiq:decred.org))
> ```
> @Guang168 can you please comment whether it was full or partial translation?

That was for full.

**Comment by xaur** [comment]
Journal Nov -> PL took 9.5-10 hrs https://matrix.to/#/!lbzTjhzNbIaDbuAxkS:decred.org/$154879875421585nLzZF:decred.org
Journal Dec -> PL took 7.5 hrs https://matrix.to/#/!lbzTjhzNbIaDbuAxkS:decred.org/$154913069426498rztun:decred.org https://github.com/artikozel/DecredJournalPL/blob/master/journal/201812_DecredJournalPL.md

**Comment by xaur** [comment]
@artikozel proposed table format below

* title on original language, links to primary source document
* target language, links to primary destination document
* effort contains time spent, links to chat message or github comment
* translators
* table must be sorted by title column for easy navigation

original title|to language|effort|translators
-|-|-|-
[Article A](http://example.com)|[PT](http://example.com)|[5 hrs](http://example.com)|user 1
[Article A](http://example.com)|[PL](http://example.com)|[4 hrs](http://example.com)|user 2, user 3
[Article B](http://example.com)|[PT](http://example.com)|[3 hrs](http://example.com), partial|user 1
[Article C](http://example.com)|[RU](http://example.com)|[10 hrs](http://example.com), diverged|user 4

source:

```
original title|to language|effort|translators
-|-|-|-
[Article A](http://example.com)|[PT](http://example.com)|[5 hrs](http://example.com)|user 1
[Article A](http://example.com)|[PL](http://example.com)|[4 hrs](http://example.com)|user 2, user 3
[Article B](http://example.com)|[PT](http://example.com)|[3 hrs](http://example.com), partial|user 1
[Article C](http://example.com)|[RU](http://example.com)|[10 hrs](http://example.com), diverged|user 4
```

**Comment by artikozel** [comment]
Yeah, this is what I had in mind, but I wanted to avoid duplicating rows for the same article, but I suppose it's inevitable since you can only have one translator per row...

**Comment by xaur** [comment]
I see no reason to have a separate table for the journal.

**Comment by xaur** [comment]
* who: @elian
* what: Decred Journal - December 2018 ([source](https://github.com/xaur/decred-news/blob/master/journal/201812.md))
* to: Spanish ([link](https://medium.com/@decred_es/revista-decred-diciembre-2018-79093f957aac))
* time: 10h ([report](https://matrix.to/#/!lbzTjhzNbIaDbuAxkS:decred.org/$1549339117763719QmRCO:matrix.org))

**Comment by xaur** [comment]
* who: `@guang`
* what: Decred Journal - January 2019 (full) ([source](https://xaur.github.io/decred-news/journal/201901.html))
* to: Chinese ([link](https://github.com/Guang168/DecredCNJournal/blob/master/201901_DecredJournalCN.md))
* time: 6h by guang + ? by contributors ([report](https://matrix.to/#/!lbzTjhzNbIaDbuAxkS:decred.org/$15506087002599CAQer:decred.org))
